RecoEgamma/ElectronIdentification/interface/ElectronMVAVariableHelper.h

```diff
@@ -0,0 +1,183 @@
+#ifndef ELECTRONMVAVARIABLEHELPER_H
+#define ELECTRONMVAVARIABLEHELPER_H
+
+#include "FWCore/Framework/interface/EDProducer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+
+#include "DataFormats/Common/interface/ValueMap.h"
+#include "DataFormats/Candidate/interface/CandidateFwd.h"
+#include "DataFormats/Candidate/interface/Candidate.h"
+
+#include "DataFormats/VertexReco/interface/Vertex.h"
+#include "DataFormats/VertexReco/interface/VertexFwd.h"
+
+#include "DataFormats/Math/interface/deltaR.h"
+
+#include "DataFormats/ParticleFlowCandidate/interface/PFCandidate.h"
+#include "DataFormats/PatCandidates/interface/PackedCandidate.h"
+
+#include "DataFormats/Candidate/interface/CandidateFwd.h"
+#include "DataFormats/Candidate/interface/Candidate.h"
+
+#include "DataFormats/PatCandidates/interface/Electron.h"
+
+#include "DataFormats/EgammaCandidates/interface/Conversion.h"
+#include "RecoEgamma/EgammaTools/interface/ConversionTools.h"
+
+#include "TMath.h"
+
+typedef edm::View<reco::Candidate> CandView;
+
+template <class ParticleType>
+class ElectronMVAVariableHelper : public edm::EDProducer {
```

Comment:
Make it a stream module, not a legacy one
(e.g.: https://indico.cern.ch/event/351556/contributions/1753858/attachments/695186/954584/multithreading.pdf)